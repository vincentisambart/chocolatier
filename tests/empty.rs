pub use chocolatier::chocolatier;

#[chocolatier]
mod objc {}

fn main() {}
