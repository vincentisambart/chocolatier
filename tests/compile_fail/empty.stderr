error: chocolatier requires at least one import!()
 --> $DIR/empty.rs:3:1
  |
3 | #[chocolatier]
  | ^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
